aws/resource_aws_rds_cluster_instance_test.go

```diff
@@ -852,13 +852,6 @@ func TestAccAWSRDSClusterInstance_CACertificateIdentifier(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSClusterDestroy,
 		Steps: []resource.TestStep{
-			{
-				Config: testAccAWSRDSClusterInstanceConfig_CACertificateIdentifier(rName, "rds-ca-2015"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSClusterInstanceExists(resourceName, &dbInstance),
-					resource.TestCheckResourceAttr(resourceName, "ca_cert_identifier", "rds-ca-2015"),
-				),
-			},
 			{
 				ResourceName:      resourceName,
 				ImportState:       true,
```

Comment:
ImportState testing cannot be the first TestStep as there is no Terraform state to reference. In the current available of the Terraform Plugin SDK, [this will cause a panic](https://github.com/hashicorp/terraform-plugin-sdk/issues/373). There is also an [open issue in `tfproviderlint` to report this problem](https://github.com/bflad/tfproviderlint/issues/134), e.g. in CI testing.

The `TestStep`s here will need to be:

- Apply rds-ca-2019 (step below this one) first
- ImportState

Reply:
Thanks, done.